clusterloader2/docs/test_suite_design.md

```diff
@@ -0,0 +1,43 @@
+# Cluster loader - TestSuite
+
+## Background
+
+Since 08/2018 cluster loader become main tool for kubernetes e2e performance tests.
+It is a very convenient tool for running single test case, but not for running
+test bundles. Currently it is impossible to run the same test
+with two different parameter sets in a single job run.
+This forces developers to create separate job configs for
+almost each test.
+
+## TestSuite
+
+TestSuite is a set of test scenarios running in a single test job.
```

Comment:
I don't think this comment brings any value :)

Reply:
`set` -> `list`. List of scenarios suggests that we will be running tests in the same order as provided.